app/javascript/react/src/components/Message.js

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  message,
+  messageStatus,
+  getMessage,
+} from '../redux/message.js';
+
+const Message = () => {
```

Comment:
- [ ] In a bid to stick to project requirements, I would implore you to change this file name to Greeting. This will guide you in future on sticking to project requirements.

Reply:
Name has been changed.